bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/models/v2/TeaserImplTest.java

```diff
@@ -96,4 +97,18 @@ protected void testTeaserWithActions() {
         Utils.testJSONExport(teaser, Utils.getTestExporterJSONPath(testBase, "teaser9"));
     }
 
+    @Test
+    protected void testEmptyTeaserImageDelegatingToFeaturedImage() {
+        Teaser teaser = getTeaserUnderTest(TEASER_20);
+        Resource imageResource = teaser.getImageResource();
+        ValueMap imageProperties = imageResource.getValueMap();
+        String linkURL = imageProperties.get("linkURL", String.class);
+        String fileReference = imageProperties.get("fileReference", String.class);
+        assertEquals("/content/teasers/jcr:content/cq:featuredimage", imageResource.getPath(), "image resource: path");
+        assertEquals("core/wcm/components/image/v2/image", imageResource.getResourceType(), "image resource: resource type");
```

Comment:
Shouldn't this be `v3` already?

Reply:
Addressed in https://github.com/adobe/aem-core-wcm-components/pull/1574/commits/fe69c8521f680287684b16203fca25fb4d822e53